content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -106,6 +108,39 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+In version 1.11, `ScheduleDaemonSetPods` was introduced as an alpha feature. With this feature,
+DaemonSet Pods are scheduled by default scheduler, instead of Daemonset controller.
+
+When this feature is enabled, the `NodeAffinity` term (instead of `.spec.nodeName`) is added to the DaemonSet Pods,
+as in the following example. This enables the default scheduler to bind the Pod to the target host. If node affinity of
+DaemonSet Pod already exists, it will be replaced. DaemonSet controller will only perform these operations when creating
+DaemonSet Pods; and those operations will only modify the Pods of DaemonSet, no changes are made to the `.spec.template`
+of DaemonSet.
+
+```
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+    - matchFields:
+      - key: metadata.name
+        operator: In
+        values:
+        - target-host-name
+```
+
+When this feature is enabled, `node.kubernetes.io/unschedulable:NoSchedule` toleration is added automatically to
+DaemonSet Pods. DaemonSet controller was ignoring `unschedulable` Nodes when scheduling DaemonSet Pods.
+In order to ensure that default scheduler keeps the same behavior and schedules DaemonSet Pods on `unschedulable` nodes,
+`TaintNodesByCondition` must be enabled.
+
+When this feature and `TaintNodesByCondition` are enabled together, `node.kubernetes.io/network-unavailable:NoSchedule`
+toleration is required to DaemonSet pods that using host network.
```

Comment:
This sentence is unclear. Could you please rephrase?

Reply:
Something like this: When this feature and `TaintNodesByCondition` are enabled together, if DaemonSet uses host network, `node.kubernetes.io/network-unavailable:NoSchedule` toleration is required. ?

Reply:
Yes, that's true. DaemonSet controller should add this toleration automatically to keep behaviour consistent. I'll open an PR for this :)